composer.json

```diff
@@ -6,23 +6,27 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/multivalue_form_element": "^1.0@beta",
         "easyrdf/easyrdf": "1.0.0 as 0.9.1",
```

Comment:
We don't need anymore this dependency (have already in rdf_skos). Can be removed as in other oe_contact_forms component.
```suggestion
```